src/Authoring/WinRT.SourceGenerator/WinRTRules.cs

```diff
@@ -18,124 +18,125 @@ private static DiagnosticDescriptor MakeRule(string id, string title, string mes
                 category: "Usage",
                 defaultSeverity: DiagnosticSeverity.Error,
                 isEnabledByDefault: true,
-                helpLinkUri: "https://docs.microsoft.com/en-us/previous-versions/hh977010(v=vs.110)");
+                helpLinkUri: "https://github.com/microsoft/CsWinRT/tree/master/src/Authoring/WinRT.SourceGenerator/AnalyzerReleases.Unshipped.md");
```

Comment:
We may want this to be AnalyzerReleases.Shipped.md, but I am not sure when we consider a version of our SourceGenerator analyzer "released"